src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -1235,3 +1278,39 @@ std::string SotaUptaneClient::secondaryTreehubCredentials() const {
 Uptane::LazyTargetsList SotaUptaneClient::allTargets() const {
   return Uptane::LazyTargetsList(images_repo, storage, uptane_fetcher);
 }
+
+void SotaUptaneClient::checkAndUpdatePendingSecondaries() {
+  // TODO: think of another alternatives to inform Primary about installation result on Secondary ECUs (reboot case)
```

Comment:
What do you mean by "reboot case"?

Reply:
This is an use-case when a reboot is required to apply an update.

Reply:
Thanks, I think I understand now.